recipes/felyx-distributed/LICENSE.md

```diff
@@ -0,0 +1,595 @@
+GNU General Public License
```

Comment:
I did not check, but this file is probably in the tarball, right? If so, you can remove it from here and conda will take care of place it in the final package accordingly.